propertysuggester/test/analyzer/test_rule_generator.py

```diff
@@ -24,7 +24,7 @@
     Entity('Q15', [
         Claim(
             Snak(31, 'wikibase-entityid', 'Q5107'),
-            [Snak(12, 'wikibase-entityid', 'Q123'), Snak(13, 'string', 'qual')],
+            [Snak(12, 'wikibase-entityid', 'Q13'), Snak(13, 'string', 'qual')],
```

Comment:
This looks like a mistake. Or was there a reason to change this?

Reply:
It was the funny hack to bring the line length under 80 chars without messing around anything (The value is not important here at all)